### PR TITLE
zipkin: Upgrading to latest official zipkin-go

### PR DESCRIPTION
**Pull request by hanochg**
### What does this PR do?
This PR upgrades the Zipkin module from 0.3.5 to 0.4.5, taking into consideration the [major refactor version 0.4.0](https://github.com/openzipkin-contrib/zipkin-go-opentracing/releases/tag/v0.4.0) had.

### Motivation
Due to a vulnerability in Apache Thrift ([CVE-2020-13949](https://lists.apache.org/thread.html/r43dc2b2e928e9d845b07ac075634cb759d91bb852421dc282f87a74a%40%3Cdev.thrift.apache.org%3E)), there's a wide need to upgrade the Thrift module to its latest version (>0.13.0).
The latest Thrift version is coupled with the latest Zipkin module, while Traefik v1.7 is using an old Zipkin module.

### More

- [x] Added/updated tests
- [x] Added/updated documentation

### Additional Notes

The new Zipkin module doesn't have a debug log anymore and it only prints error messages.
That's why I removed the `Zipkin->Debug` key Traefik configurations.

### CODE REVIEW

**Comment by hanochg** [comment]
Updated the PR, now the CI is green.

File: go.mod [hunk 1]
```diff
@@ -41,15 +41,15 @@ require (
 	github.com/emicklei/go-restful v1.1.4-0.20160814184150-89ef8af493ab // indirect
 	github.com/fatih/color v1.5.1-0.20170523202404-62e9147c64a1 // indirect
 	github.com/gambol99/go-marathon v0.7.2-0.20180614232016-99a156b96fb2
-	github.com/go-acme/lego/v3 v3.0.1
+	github.com/go-acme/lego/v3 v3.0.2
```
Comment (hanochg) [inline go.mod:44]: github.com/go-acme/lego/v3 `v3.0.1` doesn't exists anymore

**Comment by hanochg** [comment]
@jbdoumenjou kindly pinging you with this PR.
Am I missing something? from my side, it looks like it's ready for review

**Comment by jbdoumenjou** [comment]
Hello @hanochg ,
 
Thanks for your contribution!

This PR is labeled `needs-design-review`, 
as you can see in our [contribution guide](https://github.com/traefik/contributors-guide/blob/master/review_guidelines.md), this means that we have to take a closer look at the scope of impact for that PR to see, how it would interact with the other parts of Traefik.

We will come back to you soon when we did the first design review iteration.

**Comment by hanochg** [comment]
Hi @jbdoumenjou,
We didn't had any progress with the design review for the last two months. 
Kindly reminding you about it, it fixes an important security breach in v1.7.

**Comment by rtribotte** [comment]
Hello @hanochg,

Thanks again for your contribution.
We took a look at your PR and we checked that the module `github.com/apache/thrift` is effectively used by Traefik v1.7.

But by executing the command:
```
❯ go mod why -m github.com/apache/thrift
# github.com/apache/thrift
(main module does not need module github.com/apache/thrift)
```
It shows that Traefik module don't use it for compilation, and therefore Traefik should not be exposed to the CVE you mentioned.

Also, regarding the zipkin update, as the v1.7 is in a "maintenance" state, we would not accept a PR that is introducing a breaking change.

For those reasons we will decline this PR.
Let us know if you think that we miss something.

**Pull request closed without merging**